pygmt/src/plot.py

```diff
@@ -43,7 +44,8 @@
     t="transparency",
 )
 @kwargs_to_strings(R="sequence", c="sequence_comma", i="sequence_comma", p="sequence")
-def plot(self, x=None, y=None, data=None, sizes=None, direction=None, **kwargs):
+@deprecate_parameter("sizes", "size", "v0.4.0", "v0.6.0")
```

Comment:
I would use the keyword name here, make it easier to regex match when the time comes to deprecate something. Also, do we want to deprecate this at v0.6.0 or v0.5.0?

```suggestion
@deprecate_parameter("sizes", "size", "v0.4.0", remove_version="v0.6.0")
```

Reply:
> Also, do we want to deprecate this at v0.6.0 or v0.5.0?

As `sizes` is heavily used, I'd like to give users more time to change the old names to new names.